Resources/Prototypes/SimpleStation14/Loadouts/jobs.yml

```diff
@@ -105,6 +105,7 @@
   exclusive: true
   jobWhitelist:
     - MedicalDoctor
+    - ChiefMedicalOfficer
```

Comment:
The CMO coat is quite literally their most recognizable item beyond their hardsuit, and I feel allowing them to simply replace it with a regular lab coat would be foolish. Besides, theirs is _far_ shwankier.

```suggestion
```

Reply:
They have a backup in their locker
If they want a different coat they'd just replace theirs right after they joined